sphinx_immaterial/__init__.py

```diff
@@ -235,13 +246,7 @@ def html_page_context(
                 "site_name": context["docstitle"],
                 "repo_url": theme_options.get("repo_url"),
                 "repo_name": theme_options.get("repo_name", None),
-                "extra": {
-                    "version": version_config,
-                    "social": theme_options.get("social"),
-                    "disqus": theme_options.get("disqus"),
-                    "manifest": theme_options.get("pwa_manifest"),
-                    "analytics": analytics,
-                },
+                "extra": extra,
```

Comment:
Heads up: I just merged #298 which directly conflicts this diff:
https://github.com/jbms/sphinx-immaterial/blob/a805d83cb974a1d7bb9aeb68297c76eda34da223/sphinx_immaterial/__init__.py#L244

Reply:
Fixed